src/main/java/net/sacredlabyrinth/phaed/simpleclans/migrations/ConfigMigration.java

```diff
@@ -0,0 +1,18 @@
+package net.sacredlabyrinth.phaed.simpleclans.migrations;
+
+import net.sacredlabyrinth.phaed.simpleclans.managers.SettingsManager;
+import org.bukkit.configuration.file.FileConfiguration;
+
+public abstract class ConfigMigration implements Migration {
+
+    protected final SettingsManager settings;
+    protected final FileConfiguration config;
+
+    public ConfigMigration(SettingsManager settingsManager) {
+        this.settings = settingsManager;
+        this.config = settings.getConfig();
+
+        migrate();
+        settings.save();
```

Comment:
Hm, migrations execute after instantiating the SettingsManager.
Hypothetically, I should reload the config as well,
Right? 👀

Reply:
It would be needed if some default value has been changed and migration is about to change that value.
It's a rare situation and should be processed by its own.